pulser-simulation/requirements.txt

```
@@ -1,3 +1,3 @@
 qutip~=4.7.2
```

You should also change this to 
```suggestion
qutip~=4.7.5
```